code/game/machinery/canister_frame.dm

```diff
@@ -50,27 +50,7 @@
 	icon_state = "frame_1"
 
 	prev_tier = /obj/structure/canister_frame/machine/frame_tier_0
```

Comment:
This seems like more of a legacy thing then anything else, should likely get renamed.
I am a bit curious about why you're keeping the frame -> canister pipeline, any particular reason?

Reply:
I think that keeping this allows eventual expansion on different types of canisters, maybe a stasis one or a self heating one. is a good pipeline that works very well and supports many different types of constructions. Plus is better than just "use iron mats, get canister" directly

Reply:
Is this gonna be one of those "someone is gonna extend this feature I'm sure of it" -- 10 years later -- "Why the fuck is it called tier 0????"

I'm good with it in general I mostly just object to the names honestly

Reply:
lol

the name can be changed yeah